### DISCUSSION TITLE
lfs bandwith quota

### DISCUSSION THREAD

**Discussion by elfunesto** [community/community#45995] [Actions]
### Select Topic Area

LFS Quota

### Body

Hi, Since a few week, I get an error message when running github actions (project RCaNmodel):

` This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access`

However, in my billing plan, my lfs quota is still at zero. I have bought a data pack but this has not solved this issue. What is the issue?
Yours truly

**Comment by jge162** [comment]
Hello @elfunesto Verify that the data pack was purchased for the correct account and repository.

If it was then Contact -> [GitHub Support](https://support.github.com/request/account?tags=hubberfy_account)  and provide them with the repository name and the error message you are receiving.

**Comment by elfunesto** [comment]
hello @jge162, I bought with my personnal account but I fear that the repository is owned by my institute account, that would explain why it doesn't work

**Reply by jge162** [reply]
That makes sense now.
